src/main/java/com/asu/cse545/group12/dao/AuthorizationDaoImpl.java

```diff
@@ -164,34 +167,40 @@ public int forward(Authorization authorization)
 		String req_status4=Const.REJECT;
 		String req_type =Const.SIGNUP_REQUEST;
 		//WE CAN DEFINE THE TYPE OF REQUESTS THAT CAN BE SEEN BY A REGULAR USER ... AS OF NOW I DEFINED THAT HE CAN ALL TYPES OF REQ BUT CANNOT SEE "PII ACCESS" REQUESTED BY GOVT AGENCIES -- ONLY ADMIN CAN SEE THAT REQUESTS
-		String whereClause = "from authorization where request_status=:req_status1 or request_status=:req_status2 or request_status=:req_status3 or request_status=:req_status4 and request_type not like('PII Access'))";
+
+//		String whereClause = "from authorization where request_type not like('PII Access'))";
+//		String whereClause = "from authorization where request_status=:req_status1 or request_status=:req_status2 or request_status=:req_status3 or request_status=:req_status4 and request_type not like('PII Access'))";
+
+		String whereClause = "from authorization where request_type not like 'Pii Access'";
```

Comment:
We can add that..i just commented ur qry strings..i vl chnge thm bak to wht u kept once its finalized...

-----Original Message-----
From: "Nagarjuna Myla" <notifications@github.com>
Sent: ‎10/‎23/‎2015 6:25 PM
To: "chasethenag420/sbs" <sbs@noreply.github.com>
Cc: "Charan Thej Aware" <charan.1312@gmail.com>
Subject: Re: [sbs] lot changes (#104)

In src/main/java/com/asu/cse545/group12/dao/AuthorizationDaoImpl.java:
> @@ -164,34 +167,40 @@ public int forward(Authorization authorization)
>  		String req_status4=Const.REJECT;
>  		String req_type =Const.SIGNUP_REQUEST;
>  		//WE CAN DEFINE THE TYPE OF REQUESTS THAT CAN BE SEEN BY A REGULAR USER ... AS OF NOW I DEFINED THAT HE CAN ALL TYPES OF REQ BUT CANNOT SEE "PII ACCESS" REQUESTED BY GOVT AGENCIES -- ONLY ADMIN CAN SEE THAT REQUESTS
> -		String whereClause = "from authorization where request_status=:req_status1 or request_status=:req_status2 or request_status=:req_status3 or request_status=:req_status4 and request_type not like('PII Access'))";
> +
> +//		String whereClause = "from authorization where request_type not like('PII Access'))";
> +//		String whereClause = "from authorization where request_status=:req_status1 or request_status=:req_status2 or request_status=:req_status3 or request_status=:req_status4 and request_type not like('PII Access'))";
> +
> +		String whereClause = "from authorization where request_type not like 'Pii Access'";
Do you want to add Pii Access from conts.java?
—
Reply to this email directly or view it on GitHub.